crates/forge_analyzer/src/definitions.rs

```diff
@@ -1579,6 +1584,40 @@ impl Visit for FunctionCollector<'_> {
             *self.res.def_mut(owner).expect_body() = body;
         }
     }
+}
+
+impl Visit for FunctionCollector<'_> {
```

Comment:
Also, I just realized you'll have to set the owner correctly for stuff like:
`export default function() {}` where the function doesn't have a name.